esp/src/eclwatch/css/hpcc.css

```diff
@@ -1322,7 +1322,7 @@ table.miniSelect span {
     left: 0;
     right: 0;
     height: auto;
-    border-color: rgb(158, 158, 158);
+    border-style: none;
```

Comment:
Not technically needed for the Tab fix - but makes more consistent with the fluent ui version